tests/ConsoleColorTest.php

```diff
@@ -1,5 +1,5 @@
 <?php
-use JakubOnderka\PhpConsoleColor\ConsoleColor;
+use PHPParallelLint\PhpConsoleColor\ConsoleColor;
```

Comment:
Would it make sense to add a namespace to this test file as well ? I'm thinking `PHPParallelLint\PhpConsoleColor\Tests` would be in line with PSR-4 ?